e2e/cloud_manager/dbusers_test.go

```diff
@@ -11,7 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-// +build e2e cloudmanager,remote,replica
+//go:build e2e || (cloudmanager && remote && replica) || (opsmanager && remote && replica)
```

Comment:
this is the new (1.17) way of expressing build tags, IDEA is auto-adding them on save, not going to fight it